### PR TITLE
test(switch): update demo html to match Figma

### PR DESCRIPTION
**Pull request by ManahariDahal**
**Related issue**: #2365

**Summary**:
- matched the current calcite-switch demo page according to the Figma design

### CODE REVIEW

File: src/demos/calcite-switch.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!-- <!DOCTYPE html>
```
Comment (driskull) [inline src/demos/calcite-switch.html:1]: Can you remove this comment?

File: src/demos/calcite-switch.html [hunk 2]
```diff
@@ -113,4 +113,144 @@ <h3>RTL Inherited from Container</h3>
...
+
+    <!-- Importing the asset head -->
+    <script src="./_assets/head.js"></script>
+
+    <style>
+      .main-container {
+        margin: 0 auto;
+        display: grid;
+        grid-template-columns: repeat(5, 1fr);
+        /* justify-content: center; */
```
Comment (driskull) [inline src/demos/calcite-switch.html:134]: Can you remove commented out code?

File: src/demos/calcite-switch.html [hunk 3]
```diff
@@ -113,4 +113,144 @@ <h3>RTL Inherited from Container</h3>
...
+    </div>
+
+    <!-- RTL Set directly on Switch -->
+    <div class="switch-change" style="margin: 90px 0 0 20%">
+      <p>RTL Set Directly on Switch</p>
+      <p>
+        <calcite-switch dir="rtl"> </calcite-switch>
+      </p>
+
+      <p style="padding: 90px 0 0 0" ;>Listening for "calciteSwitchChange"</p>
```
Comment (driskull) [inline src/demos/calcite-switch.html:242]: Can we add a class for these inlined styles?

**Review by driskull — COMMENTED** [review]
👍

**Comment by bstifle** [comment]
image

let's drop these labels down to avenir-medium

**Review by julio8a — APPROVED** [review]
This looks good! The only thing I noticed was that clicking 'disabled' enabled the already disabled switches. But I don't think this should stop us from merging this one in. (I think this might be the case for other components in demo pages as well)

**Comment by bstifle** [comment]
@ManahariDahal 

would like to get the responsiveness ironed out.

the top section shouldn't be centered, but left aligned.

image


also, let's be accurate with the labels, can we change the last column to "Switched Disabled"

**Review by bstifle — APPROVED** [review]
lookin good

**Comment by julio8a** [comment]
Travis CI failed. @jcfranco, can you guide @ManahariDahal on how to fix?
Screener is showing a diff in rating, which code was not touched in this PR, which is odd.

**Comment by driskull** [comment]
Can you try merging in latest master and running again?

**Comment by ManahariDahal** [comment]
@julio8a @driskull It passes the checks. Thank you for the help.

**Comment by driskull** [comment]
If it fails sometimes you just gotta reboot it :D

**Comment by ManahariDahal** [comment]
> If it fails sometimes you just gotta reboot it :D

will keep it in mind :D

**Pull request merged**